charters/CommunicationsTeam/charter.md

```diff
@@ -0,0 +1,64 @@
+
+# Data Coordination Platform Communications Team
+
+
+## Description
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) Communications team coordinates and publishes public communications through various channels for the DCP as a whole. We work with the different groups within the DCP to ensure that each group is properly represented in external communications. We work with the HCA communications representatives if/when needed to ensure that DCP communications are consistent with and support those of the HCA. We monitor the DCP Data Portal documentation and update as needed. We develop training materials and events to teach people about the HCA DCP and how to use it.
```

Comment:
Suggested change to the sentence that starts "We work with the HCA communications representatives as needed.."  to the following:
We work with the HCA communications representatives as needed to ensure that DCP communications are distinct from but consistent with those of the HCA, and that we are sharing resources when possible and appropriate to avoid duplication of efforts and user confusion."

Reply:
I think that sounds great. I can go ahead and edit that sentence.